omod/src/main/java/org/openmrs/module/idgen/rest/resource/IdentifierSourceResource.java

```diff
@@ -278,6 +448,29 @@ public void purge(IdentifierSource identifierSource, RequestContext context) thr
         Context.getService(IdentifierSourceService.class).purgeIdentifierSource(identifierSource);
     }
     
+    @Override
+    protected PageableResult doGetAll(RequestContext context) throws ResponseException {
+        return new NeedsPaging<IdentifierSource>(Context.getService(IdentifierSourceService.class).getAllIdentifierSources(false), context);
+    }
+    
+    @Override
+    protected PageableResult doSearch(RequestContext context) throws ResponseException {
+        String identifierType = context.getRequest().getParameter("identifierType");
+        if(identifierType != null){
+            PatientIdentifierType requestedPatientIdentifierType = Context.getPatientService().getPatientIdentifierTypeByUuid(identifierType);
+            if(requestedPatientIdentifierType == null)
```

Comment:
For this, did you take a look at? https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-CodeStyle

Reply:
fixed. added curly braces.